lib/evss/vso_search/service.rb

```diff
@@ -29,12 +29,14 @@ def get_current_info(additional_headers = {})
       private
 
       def request_headers(additional_headers)
-        ssn = additional_headers.key?('va_eauth_pnid') ? additional_headers['va_eauth_pnid'] : @user.ssn
-        edipi = additional_headers.key?('va_eauth_dodedipnid') ? additional_headers['va_eauth_dodedipnid'] : @user.edipi
+        ssn = additional_headers['va_eauth_pnid'].presence || @user.ssn
+        edipi = additional_headers['va_eauth_dodedipnid'].presence || @user.edipi
         if edipi.nil?
           user_info = {
             loa3: @user.loa3?,
-            mhv_icn: !@user.mhv_icn.nil?
+            mhv_icn: !@user.mhv_icn.nil?,
+            edipi_type: additional_headers['va_eauth_dodedipnid'].class,
+            edipi_length: additional_headers['va_eauth_dodedipnid'].class.length
```

Comment:
I think this line will always result in an undefined method error. Should it maybe be:

`edipi_length: additional_headers['va_eauth_dodedipnid']&.length`